adapter/go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/wso2/apk/adapter
 go 1.19
 
 require (
-	github.com/envoyproxy/go-control-plane v0.10.3
+	github.com/envoyproxy/go-control-plane v0.10.3-0.20230119052701-0ebb8a16ea22
```

Comment:
shall we update to latest go control plane version?

Reply:
0.11.0

Reply:
fixed